### ISSUE TITLE
Excluir serviço é enviar exclusão para aprovação

### ISSUE THREAD

**Issue by kchnr** [servicosgovbr/editor-de-servicos#250]
O botão de excluir serviço na lista de serviços não deixa claro que a "exclusão" tem que ser aprovada para então ser de fato excluido.

Do jeito que está hoje, dá a impressão que seria excluido no momento que a lixeira for clicada.

Essa funcionalidade depende de enviar alterações para aprovação.

**Comment by sahrosa** [comment]
Talvez faça mais sentido simplificarmos, deixando apenas quem tem permissão de aprovar uma publicação, também excluir um serviço – sem passar por "fila de exclusão"

O que acham?

**Comment by cv** [comment]
Acho que não funciona muito bem, @sahrosa – concentra muito do trabalho em uma pessoa só, IMHO. Acho melhor sinalizar o pedido de remoção para quem não é publicador na UI de acordo do que concentrar o trabalho todo só no publicador.

**Comment by sahrosa** [comment]
Mas como essas mesmas pessoas vão ter que aceitar a exclusão, não fica quase na mesma que só essas pessoas excluirem? Vai ter gargalo nela igual, né? E assim iria diminuir a complexidade na UI.

Se apenas quem pode aprovar visualizar o botão de excluir, teríamos:
* botão de excluir

Se todos puderem excluir, teremos:
* mostrar o botão pra todos
* ao clicar em excluir, avisar que alguém irá aprovar a exclusão
*aprovar exclusão
*negar exclusão
* notificar que o serviço foi efetivamente excluído
* notificar que o serviço não foi excluído

**Comment by cv** [comment]
Concordo, @sahrosa - vamos deixar que apenas os publicadores possam excluir serviços. Sugestões e discussão sobre quais serviços excluir vão acabar acontecendo fora da plataforma do editor, de qualquer maneira...

Assim esse cartão fica bem menor :)

**Comment by sahrosa** [comment]
Ao clicar em excluir na fila de serviços, está sendo mostrado um botão "publicar"

**Comment by gustavo-freitas** [comment]
Testes unitários (remover serviço não publicado e remover serviço publicado) e manuais foram realizados para cobrir essa nova funcionalidade. Entretanto, ainda precisamos de uma abordagem mais alto nível para as nossa cobertura de testes, impedindo que funcionalidades já existentemente parem de funcionar quando introduzimos novas funcionalidades.
